app/src/Components/InteractionBar.js

```diff
@@ -1,31 +1,42 @@
-import React from "react";
+import React, { useState } from "react";
 import "./InteractionBar.css";
 
 const InteractionBar = () => {
+  const [commentCount, setCommentCount] = useState(1);
+  const [retweetCount, setRetweetCount] = useState(5);
+  const [likeCount, setLikeCount] = useState(100);
+  const [shareCount, setShareCount] = useState(10);
+
```

Comment:
The Tweet component will have the count data init, maybe the interactionBar could except those as props?